imports/plugins/core/accounts/server/no-meteor/getShopIdByDomain.js

```diff
@@ -1,12 +1,12 @@
 import url from "url";
 
-export default async function getShopIdByDomain(collections) {
+export default async function getShopIdByDomain(context) {
```

Comment:
I changed the parameter here to inspect the `context`, which should include the `rootURL`